backend/src/flotilla/api/robots_api.py

```diff
@@ -41,20 +60,70 @@ async def get_robots(db: Session = Depends(get_db)) -> List[Robot]:
     return robots
 
 
+@router.post(
+    "/robots",
+    responses={
+        HTTPStatus.OK.value: {
+            "model": CreateRobotResponse,
+            "description": "Successfully added robot to database",
+        },
+        HTTPStatus.UNAUTHORIZED.value: {
+            "model": Error,
+            "description": "Unauthorized",
+        },
+    },
+    tags=["Robots"],
+    summary="Create a new robot",
+    description="""### Overview
+    Stop the execution of the current active mission.
+    If there is no active mission on robot, nothing happens.""",
+    dependencies=[Security(authentication_scheme)],
+)
+async def post_robot(
+    db: Session = Depends(get_db),
+    robot: Robot = Body(None, description="New robot to be added"),
+) -> dict:
+    try:
+        robot_id: int = create_robot(
+            db=db,
+            name=robot.name,
+            model=robot.model,
+            serial_number=robot.serial_number,
+            host=robot.host,
+            port=robot.port,
+            enabled=robot.enabled,
+            capabilities=robot.capabilities,
+        )
+    except HTTPException as e:
+        logger.exception(f"An error occurred while creating a robot: {e.detail}")
+        raise
+
+    return {"robot_id": robot_id}
```

Comment:
Why list a model in the description when it is not used here? Guess they are the same, but a bit weird